doc/qgsquick.dox

```diff
@@ -13,6 +13,8 @@ QGIS Quick consists of a Qt plugin that provides the QML components and of a sha
 
 \subsection qgsquick_overview_widgets QML Classes
 \subsubsection qgsquick_overview_widgets_mapcanvas MapCanvas
+\subsubsection qgsquick_overview_widgets_scalebar ScaleBar
+\subsubsection qgsquick_overview_widgets_messagelog MessageLog
```

Comment:
when docs are built, does this links to `QgsQuickMessageLog` class documentation? I am not sure if this was meant to have a short description in each subsection or just a link to the implementation..? (QML + C++ class?)